src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -444,6 +491,25 @@ private static boolean hasLineBreakBefore(DetailAST rightCurly) {
     }
 
     /**
+     * Checks if right curly is alone on line in token the lambda.
+     * @param details for validation.
+     * @return true, if right curly is alone on line.
+     */
+    private static boolean isAloneLambda(Details details) {
```

Comment:
This was the same comment I made before [here](https://github.com/checkstyle/checkstyle/pull/3680#discussion_r95815987).